bindgen/ir/context.rs

```diff
@@ -2159,8 +2159,8 @@ If you encounter an error missing from this list, please file an issue or a PR!"
         let mut kind = ModuleKind::Normal;
         let mut looking_for_name = false;
         for token in cursor.tokens().iter() {
-            match token.spelling() {
-                b"inline" => {
+            match token.spelling().to_str().unwrap() {
```

Comment:
What guarantees that the tokens are utf-8? I think the previous code was fine.

Reply:
We already assume UTF-8 in a bunch of other places, e.g. macro names and variable names. Granted here it's more for consistency rather than necessity.